file-formats/aifa/README.md

```diff
@@ -0,0 +1,11 @@
+# Action File Format
```

Comment:
Observations related to recent README updates: @schneidermic0 this section title is generated from the current SVAL object. Merging the pr then may lead to uncorrect README as the SVAL will be checked much later in our process.

Reply:
Yeah, on the other hand, I think it is helpful to see a meaningful name (even if it is not 100% correct and might be updated in the future) instead of having none.